cookbooks/fb_helpers/libraries/node_methods.rb

```diff
@@ -689,7 +689,7 @@ def root_user
 
     def root_group
       value_for_platform(
-        %w{openbsd freebsd mac_os_x macos} => { 'default' => 'wheel' },
```

Comment:
@rb2k can you check that this is fine on bigsur too?

Reply:
It depends on which version of chef you run.
Newer versions will map macOS to always be mac_os_x:

https://github.com/chef/ohai/blob/f26c3ce26c63eb06d714073f920b1f75c4feef25/lib/ohai/plugins/darwin/platform.rb#L35

older versions  (e.g. chef 14) will not:
https://github.com/chef/ohai/blob/14-stable/lib/ohai/plugins/darwin/platform.rb#L30

This was the PR: https://github.com/chef/ohai/pull/1470